### ISSUE TITLE
NumberSpan wrong implementation

### ISSUE THREAD

**Issue by darienalvarez** [SufficientlySecure/html-textview#166]
Hello:

NumberSpan implementation is leading to an exception:

```
Fatal Exception: java.lang.AbstractMethodError

abstract method "int android.text.ParcelableSpan.getSpanTypeIdInternal()"

android.view.accessibility.AccessibilityNodeInfo.writeToParcel
```

you may find a good explanation of the problem here:

> https://medium.com/@numan1617/android-framework-classes-choose-wisely-2f8300f6c255#.3669g6u0m

**Comment by darienalvarez** [comment]
I did a PR for you guys, I will appreciate if you are able to merge it:

https://github.com/SufficientlySecure/html-textview/pull/167

**Comment by naushad-madakiya** [comment]
@dschuermann  @darienalvarez close the issue if it's fixed.

**Comment by farimarwat** [comment]
same issue. kindly fix it

java.lang.AbstractMethodError: 
  at android.text.TextUtils.writeToParcel (TextUtils.java:702)
  at android.os.Parcel.writeCharSequence (Parcel.java:729)
  at android.view.accessibility.AccessibilityNodeInfo.writeToParcelNoRecycle (AccessibilityNodeInfo.java:3372)
  at android.view.accessibility.AccessibilityNodeInfo.writeToParcel (AccessibilityNodeInfo.java:3181)
  at android.os.Parcel.writeTypedObject (Parcel.java:1516)
  at android.os.Parcel.writeTypedList (Parcel.java:1395)
  at android.os.Parcel.writeTypedList (Parcel.java:1380)
  at android.view.accessibility.IAccessibilityInteractionConnectionCallback$Stub$Proxy.setFindAccessibilityNodeInfosResult (IAccessibilityInteractionConnectionCallback.java:144)
  at android.view.AccessibilityInteractionController.updateInfosForViewportAndReturnFindNodeResult (AccessibilityInteractionController.java:869)
  at android.view.AccessibilityInteractionController.findAccessibilityNodeInfoByAccessibilityIdUiThread (AccessibilityInteractionController.java:340)
  at android.view.AccessibilityInteractionController.access$400 (AccessibilityInteractionController.java:67)
  at android.view.AccessibilityInteractionController$PrivateHandler.handleMessage (AccessibilityInteractionController.java:1324)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6898)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:537)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)

**Comment by darienalvarez** [comment]
The issue was fixed, but has not being released yet, so try to fork the project and do your own library until they release a new version of this library

**Comment by dschuermann** [comment]
released in 3.7
